### ISSUE TITLE
package lsp-mode-booster in melpa

### ISSUE THREAD

**Issue by ericdallo** [blahgeek/emacs-lsp-booster#32]
Have you ever considered moving the advice content to a package for easy install/setup? that would make simpler to configured in lsp-mode and maybe we could ship it in doom-emacs and related for easier spread this package.

**Comment by jdtsmith** [comment]
> I noticed that the eglot-lsp-booster emacs package is on melpa

Nope, not yet.  I'd probably want a better implementation before doing that.
